packages/pods-core/src/v2/pods/export/MarkdownExportPodV2.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  DEngineClient,
+  IntermediateDendronConfig,
+  NoteProps,
+} from "@dendronhq/common-all";
+import {
+  DendronASTDest,
+  MDUtilsV4,
+  MDUtilsV5,
+  RemarkUtils,
+} from "@dendronhq/engine-server";
+import { JSONSchemaType } from "ajv";
+import _ from "lodash";
+import {
+  ExportPodV2,
+  MarkdownV2PodConfig,
+  RunnableMarkdownV2PodConfig,
+} from "../../..";
+
+/**
+ * Markdown Export Pod (V2 - for compatibility with Pod V2 workflow). Supports
+ * both exportText() and exportNote()
+ */
+export class MarkdownExportPodV2 implements ExportPodV2<string> {
+  private _config: RunnableMarkdownV2PodConfig;
+  private _engine: DEngineClient;
+  private _dendronConfig: IntermediateDendronConfig;
+
+  constructor({
+    config,
+    engine,
+    dendronConfig,
+  }: {
+    config: RunnableMarkdownV2PodConfig;
+    engine: DEngineClient;
+    dendronConfig: IntermediateDendronConfig;
+  }) {
+    this._config = config;
+    this._engine = engine;
+    this._dendronConfig = dendronConfig;
+  }
+
+  async exportText(_input: string): Promise<string> {
+    const proc = MDUtilsV5.procRemarkParseNoData(
+      {},
+      { dest: DendronASTDest.MD_REGULAR }
+    );
+
+    // TODO: markdown process utils don't currently take raw text; requires NoteProps for all the util methods.
```

Comment:
can we remove this comment?

Reply:
I left it there intentionally to mark where we need to add implementation to make `exportText()` actually work.